frontend/src/js/route-config/route-config.js

```diff
@@ -589,8 +597,6 @@
         });
 
         $rootScope.$on('$stateChangeSuccess', function() {
-            // Save the route title
-            $rootScope.pageTitle = $state.current.title;
```

Comment:
why are we removing this?

Reply:
ngMeta also updates the title of the page